CHANGELOG_PENDING.md

```diff
@@ -1,3 +1,25 @@
 # Unreleased Changes
 
-## vX.X
+## v0.5.16
```

Comment:
Fine to leave this as vX.X as we'll add the release version to `CHANGELOG` when the release is cut.